index.html

```diff
@@ -36,3 +39,134 @@ <h2 class="support-text">I’am a Software Developer</h2>
             <li class="links"><a href="#"><img src="images/Vector3.svg" alt="Media"></a></li>
         </ul>
     </section>
+    <section class="grid-container">
+        <h2 class="title">My Recent Works</h2>
+        <div class="image-line1"><img src="images/line1.svg" alt=""></div>
+        <div class="card-1">
+            <div class="image"><img src="images/imagecard1.png" alt=""></div>
+            <h3 class="multipost-h">Multi-Post Stories</h3>
+            <p class="para1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="tags">
+                <li class="csbox">CSS</li>
+                <li class="htmlbx">HTML</li>
+                <li class="bootbox">Bootstrap</li>
+                <li class="rubox">Ruby</li>
+            </ul>
+            <button class="button1">See Project</button>
+        </div>
+    </section>
+
+    <section class="grid-container2">
+        <div class="project-bg">
+            <h3 class="professional-heading">Profesional Art Printing Data</h3>
+            <p class="daily-section">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="socials">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="button2">See Project</button>
```

Comment:
- [ ] According to the requirements, You should Implement the button interactions (enable, hover, etc.). Kindly, add the button interactions based on your Figma Design 👌.

![1](https://user-images.githubusercontent.com/45231276/176833382-1d1d826d-6c66-47ac-bec8-36427809918f.PNG)

![3](https://user-images.githubusercontent.com/45231276/176980454-8c16e5d0-84f5-421f-84ee-cf5d3f47c451.PNG)

- [ ] Note: you should to do this on all buttons 👌